From a5b03ff37de1c0f0d1db38d4e9a23361f0ed9aa0 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Tue, 23 Dec 2025 15:22:01 -0500
Subject: [PATCH 461/473] amdxc: Add env to enable MLFG watermark.

technically is unnecessary but certainly makes it a bit easier to set this env.
---
 dlls/amdxc64/main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dlls/amdxc64/main.c b/dlls/amdxc64/main.c
index 870d7e5eaff..45ab93ba13b 100644
--- a/dlls/amdxc64/main.c
+++ b/dlls/amdxc64/main.c
@@ -604,9 +604,9 @@ BOOL WINAPI DllMain(HINSTANCE inst, DWORD reason, void *reserved)
         case DLL_PROCESS_ATTACH:
         {
             if ((env = getenv("FSR4_WATERMARK")) && !strcmp(env, "1"))
-            {
                 _putenv("MLSR-WATERMARK=1");
-            }
+            if ((env = getenv("MLFG_WATERMARK")) && !strcmp(env, "1"))
+                _putenv("MLFI-WATERMARK=1");
             break;
         }
         default: break;
-- 
2.52.0

